internal/testprotos/desc_test_oneof.proto

```diff
@@ -1,6 +1,9 @@
 syntax = "proto3";
+
 package testprotos;
 
+option go_package = "github.com/jhump/protoreflect/internal/testprotos";
```

Comment:
Oops. The `make generate` task was broken because this file was missing a Go package.